src/RulesDeploy.ts

```diff
@@ -81,22 +86,22 @@ RulesDeploy.prototype = {
             clc.bold(resourceName)
         );
       });
-  },
+  }
 
-  _compileRuleset: function(filename, files) {
+  _compileRuleset(filename: any, files: any): Promise<any> {
```

Comment:
`_` is an indicator of `private`, so this should be `private compileRuleset` (and any calls to it adjusted)

Reply:
done